image-customization.lua

```diff
@@ -0,0 +1,169 @@
+-- Generic features and packages for all devices
+features({
+	'autoupdater',
+	'config-mode-domain-select',
+	'config-mode-geo-location-osm',
+	'ebtables-filter-multicast',
+	'ebtables-filter-ra-dhcp',
+	'ebtables-limit-arp',
+	'ebtables-source-filter',
+	'mesh-batman-adv-15',
+	'mesh-vpn-fastd-l2tp',
+	'radvd',
+	'radv-filterd',
+	'respondd',
+	'status-page',
+	'web-advanced',
+	'web-logging',
+	'web-private-wifi',
+	'web-wizard',
+})
+
+packages({
+	'ca-bundle',
```

Comment:
```suggestion
```
`ca-bundle` is part of the tls feature https://github.com/freifunk-gluon/gluon/commit/683defd0f8597773f1f3f1fb2031cb010c82ab56